include/TPP/Dialect/Tpp/TppUtils.h

```diff
@@ -75,6 +78,12 @@ bool isValConstZero(Value val);
 // Returns true if the op defining `val` represents a zero filled tensor.
 bool isZeroTensor(Value val);
 
+// Splits fused op into its individual components and returns created
+// operations.
+// The passed fused op remains untouched.
```

Comment:
Add a FIXME/TODO here to explain why this shouldn't be permanent not a pattern to repeat.